controller/rest/crdsecurityrule.go

```diff
@@ -3358,6 +3358,15 @@ func handlerGroupCfgExport(w http.ResponseWriter, r *http.Request, ps httprouter
 
 		// export process and file profiles
 		exportProcessRule(gname, &rconf, &(resptmp.Spec), acc)
+		if utils.DoesGroupHavePolicyMode(gname) {
+			allowed := utils.NewSetFromStringSlice([]string{share.PolicyModeLearn, share.PolicyModeEvaluate, share.PolicyModeEnforce})
+			for _, mode := range []*string{resptmp.Spec.Target.PolicyMode, resptmp.Spec.ProcessProfile.Mode} {
+				if mode != nil && !allowed.Contains(*mode) {
+					restRespError(w, http.StatusBadRequest, api.RESTErrInvalidRequest)
```

Comment:
If I understand the logic correctly, when hitting the logic here, the caller is asking an export data for an group that contains invalid mode, which can't be exported, is that correct?  If so, I would suggest adding some error message in the payload at least, for example:
```suggestion
					restRespErrorMessage(w, http.StatusBadRequest, api.RESTErrInvalidRequest, "the group XXX can't be exported because xxx")
```